ingest/.vscode/launch.json

```diff
@@ -0,0 +1,24 @@
+{
+    // Use IntelliSense to learn about possible attributes.
+    // Hover to view descriptions of existing attributes.
+    // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
```

Comment:
Seems like kind of a weird comment - advertising?

Reply:
this is vscode stuff, that i included as is with the code base